packages/marathon/extra/marathon.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -euo pipefail
+
+export LIBPROCESS_IP=$($MESOS_IP_DISCOVERY_COMMAND)
+
+: ${MARATHON_HOSTNAME="$LIBPROCESS_IP"}
```

Comment:
I would personally opt for more verbosity here, like:
```
[ -z "$MARATHON_HOSTNAME" ] && MARATHON_HOSTNAME="$LIBPROCESS_IP"
```

There is also a minor shell security warning when using the `: ${..}` idiom : https://github.com/koalaman/shellcheck/wiki/SC2223

Reply:
:) That's a good point. The attack (setting an env to */*/*/*/*) seems like a pretty big stretch, given that if people have the ability to attack a cluster by setting env variable values then there are way worse problems.

I do like the short-hand notation, but would be open to the `-z` notation also.

At this point, since the tests all pass an getting this PR to the state where it is passed, accepted, etc. has been such a long uphill battle, I vote that we merge as is.